README.md

```diff
@@ -1,9 +1,9 @@
-# Nictiz Generate TestScripts
+# Nictiz TestScripts NTS sources
 
-This folder contains source files for generating TestScripts from NTS-files.
+This repository contains the NTS source files for generating the TestScripts in the [Nictiz-STU3-testscripts repository](https://github.com/Nictiz/Nictiz-STU3-testscripts). NTS is a [custom shorthand format](https://github.com/Nictiz/Nictiz-tooling-testscripts/tree/main/generate) for generating HL7 FHIR TestScript resources.
 
 The NTS-files and resources referenced are placed per 'project' (information standard and version) in `src/[projectname]`. NTS components that span multiple projects are placed in `src/common-asserts`. For more info on creating NTS projects, please see [the NTS documentation](https://github.com/Nictiz/Nictiz-tooling-testscripts/tree/main/generate). The build process sources its tooling materials from this repository (this can be overridden by passing `-Dtestscripttools.dir=/path/to/testscript/tools` to the build if needed).
 
-The TestScripts can be generated by running the [ANT](https://ant.apache.org/) build or (on Windows) by execuiting `build.bat` in the project directory, or alternatively the common build from this project. When asked, fill in the relevant projectname or pass it as a parameter by using `-Dproject=[projectname]`.
+The TestScripts can be generated by running the [ANT](https://ant.apache.org/) build or (on Windows) by execuiting `build.bat` in the project directory.
 
-The output is placed in the `build/[projectname]` folder and is not committed to the repository at this location. Please check and copy the output to it's intended location by hand. If you are absolutely certain that a build can be copied, the ANT `-Dbuild.dir` property can be set. At this location, a `[projectname]` folder with the output in it will still be created. Note that `${build.dir}/[projectname]` is deleted before creating new output, please make sure there are only generated TestScripts in this directory.
\ No newline at end of file
+The output is placed under the `dev` folder and is committed together with the sources. This folder happens to be a git subtree of the Nictiz-STU3-testscripts repository. When the output is stable, a `git subtree push` can be performed to synchronize the final output with this repository.
```

Comment:
Dit mag misschien nog wat meer toegelicht worden? Kan ook d.m.v. een stappenplan in het Kwaliteitshandboek bijv.

Reply:
Zeker. Dit was eigenlijk meer placeholder text.

Reply:
Ik heb eea. in het kwaliteitshandboek neergepend.